src/default.css

```diff
@@ -1,3 +1,7 @@
 @tailwind base;
 @tailwind components;
-@tailwind utilities;
\ No newline at end of file
+@tailwind utilities;
+
+.widget-shadow {
```

Comment:
This might be a good thing to use a styled-component for instead of adding to the CSS here. Let me know if you need help with that, but the docs are [here](https://styled-components.com/)